### DISCUSSION TITLE
how to annotate intraoral 3d Scans in stl file foramt with monai

### DISCUSSION THREAD

**Discussion by abdelkareemkobo** [Project-MONAI/MONAILabel#1641] [Q&A]
I have a  data folder contain real life intra-oral 3D scans from multiple subjects in multiple positions. 
In the validation data folder can be find real life 3D scans but the rotation was made by blender (software) using multiple artificial axes and rotation angles. 
This validation data can be used for validation any kind of rotation axes finder algorithm. 
The axes and the rotation angles can be find in the algo_prec.csv file for each artificial rotations.
![image](https://github.com/Project-MONAI/MONAILabel/assets/72500236/9979da81-6e7b-444c-9fad-49cf8796449e)

**Comment by diazandr3s** [comment]
Hi @abdelkareemkobo,

From what I can see these are meshes, right? How would you create the ground truth for these meshes?

Please provide more details of what you expect from an annotation model.

**Reply by abdelkareemkobo** [reply]
Thanks @diazandr3s for response, here is the link for the [data](https://www.kaggle.com/datasets/ttorpe/intraoral-3d-scanner-dataset/data) 
yes they are meshes. 
i should annotate the tooth for  with my self!  i think before build the DL pipeline.

**Reply by diazandr3s** [reply]
Hi @abdelkareemkobo,

Thanks for the details. 

Current pre-trained models available in MONAI Label work on volumes, not meshes I'm afraid.
It'd be interesting though to have a model for that. Would you be interested in taking this initiative?

Thanks!

**Reply by abdelkareemkobo** [reply]
This sounds great. Currently, I'm still in the process of learning about 3D and meshes. After some time, once I've gained more knowledge and trained more models, I will certainly consider contributing to it. Actually, I discovered MONAI through vfdev-5, the maintainer of Ignite. Thank you!

**Reply by lassoan** [reply]
You can try the intraoral scan segmentation tools in this 3D Slicer extension: https://github.com/DCBIA-OrthoLab/SlicerAutomatedDentalTools

The mesh segmentation works well, but the issue with it is that it uses pytorch3d, a poorly engineered/supported package (for example, there are no wheels for Windows). It would be nice if MONAI included a similar model.
